engine/frontend/src/component/ServiceInfo.js

```diff
@@ -30,32 +30,43 @@ const ServiceInfo = () => {
     const params = useParams()
     const {name: enclaveName, uuid: serviceUuid} = params;
 
-    useEffect(() => {
-        let stream;
-        const ctrl = new AbortController();
+    const startServiceLogCollector = (ctrl) => {
+        let stream
         const fetchLogs = async () => {
-            stream = await getServiceLogs(ctrl, enclaveName, serviceUuid, appData.apiHost);
+            stream = await getServiceLogs(ctrl, enclaveName, serviceUuid, appData.apiHost, true, 6000);
             try {
                 for await (const res of stream) {
                     const log = res["serviceLogsByServiceUuid"][serviceUuid]["line"][0]
-                    setLogs(logs => [...logs, log])
+                    if (log !== "") {
```

Comment:
Nice check although backend shouldn't be sending empty log lines. If you're still seeing this happens a lot, let's chat via slack.